### ISSUE TITLE
Can't view affix description

### ISSUE THREAD

**Issue by duraz0rz** [domainzero/mythicpl.us#139]
Hey, I like coming to your site to view affix descriptions because I always forget what they are, but I can't seem to view them today.

Opening up the dev console gives me this error:

```
Uncaught TypeError: Cannot read property 'classList' of null
    at highlightCurrentAffixes (?g=js:1)
    at XMLHttpRequest.xhr.onreadystatechange (?g=js:2)
```

**Comment by domainzero** [comment]
Hey @duraz0rz 

This should be fixed now. Had a typo somewhere!

**Comment by domainzero** [comment]
Fixed in ad91d5a

**Comment by duraz0rz** [comment]
Looks good...thanks!
